Application/EdFi.Admin.DataAccess/Providers/AdminDatabaseConnectionStringProvider.cs

```diff
@@ -4,20 +4,42 @@
 // See the LICENSE and NOTICES files in the project root for more information.
 
 #if NETSTANDARD
+using System;
 using EdFi.Ods.Common.Configuration;
+using EdFi.Ods.Common.Context;
 
 namespace EdFi.Admin.DataAccess.Providers
 {
     public class AdminDatabaseConnectionStringProvider : IAdminDatabaseConnectionStringProvider
```

Comment:
This project generates a nuget package, and is also consumed by other tools such as Admin App. Adding these Ods specific dependencies will cause some trouble for that use case. Most likely Ods would need to create its own new version of IAdminDatabaseConnectionStringProvider implemented with this new support, and then register that for ODS/API, leaving the original unmodified.